### PR TITLE
frontend-CugFix-RideCompleted

### PR DESCRIPTION
**Pull request by ROHIT-43**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description

Added a crousel on Ride Completed Screen and Fixed some bugs

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## How did you test it?

Manually

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added integration tests for my changes where possible
- [ ] I tested the changes using integration tests
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

**Review by witcher-shailesh — APPROVED** [review]

File: Frontend/ui-customer/src/Screens/RideBookingFlow/RiderRideCompletedCard/View.purs [hunk 1]
```diff
@@ -307,6 +350,15 @@ rideDetailsButtonView state push =
 
 bottomCardView :: forall w. RiderRideCompletedScreenState -> (Action -> Effect Unit) -> PrestoDOM (Effect Unit) w
 bottomCardView state push =
+  let customerIssue :: CustomerIssue
+      customerIssue = {
+        currentIndex : state.customerIssue.currentPageIndex
+      , currentPageIndex : state.customerIssue.currentPageIndex
+      , bannerComputedView : state.customerIssue.bannerComputedView
+      , customerIssueCards : spy "issueReportBannerConfigs" $ issueReportBannerConfigs state
```
Comment (prakharritik) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/RiderRideCompletedCard/View.purs:358]: remove spy

File: Frontend/ui-customer/src/Services/Backend.purs [hunk 2]
```diff
@@ -573,8 +573,8 @@ makeFeedBackReq rating rideId feedback wasOfferedAssistance = FeedbackReq
     ,   "nightSafety" : Nothing
     }
 
-makeFeedBackReqs :: Int -> String -> String -> Maybe Boolean -> String -> FeedbackReq
-makeFeedBackReqs rating rideId feedback favDriver audio = FeedbackReq
+makeFeedBackReqs :: Int -> String -> String -> Maybe Boolean -> String -> Maybe Boolean -> Maybe Boolean -> FeedbackReq
+makeFeedBackReqs rating rideId feedback favDriver audio nightSafety wasOfferedAssistance = FeedbackReq
```
Comment (prakharritik) [inline Frontend/ui-customer/src/Services/Backend.purs:577]: Looks like we are still not using this, we are passing nothing in the request

File: Frontend/ui-driver/src/Resource/Localizable/BN.purs [hunk 3]
```diff
@@ -1604,7 +1604,7 @@ getBN stringKey =
         PET_FRIENDLY -> "পোষ্য-বান্ধব"
         FROM_WHERE -> "জন্মস্থান"
         DRIVING_SINCE -> "ড্রাইভিং শুরু"
-        WHY_NY -> "আমি কেন অটো চালানোর সিদ্ধান্ত নিলাম?"
+        WHY_NY variant -> "আমি কেন "<> (if variant == "AutoCategory" then getBN AUTO else getBN CAB) <>" চালানোর সিদ্ধান্ত নিলাম?"
```
Comment (prakharritik) [inline Frontend/ui-driver/src/Resource/Localizable/BN.purs:1607]: Lets not do this like this, we can make a function to get variant name and pass it wherever we are using this constructor. Other cases like bike ambulance are also there

File: Frontend/ui-customer/src/Screens/RideBookingFlow/RiderRideCompletedCard/ComponentConfig.purs [hunk 4]
```diff
@@ -28,5 +33,64 @@ primaryButtonConfig state = PrimaryButton.config
...
+    , rippleColor = Color.rippleShade
+    , isClickable = true
+    }
+
 driverInfoCardConfig :: RiderRideCompletedScreenState -> FavouriteDriverInfoCard.FavouriteDriverInfoCardState
-driverInfoCardConfig state = FavouriteDriverInfoCard.config 
\ No newline at end of file
+driverInfoCardConfig state = FavouriteDriverInfoCard.config 
+
+issueReportBannersTransformer  :: Array CustomerIssueCard -> Array (Record IssueReportBannerProps)
```
Comment (prakharritik) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/RiderRideCompletedCard/ComponentConfig.purs:55]: Can you confirm if these functions are present at two places? If so We can remove from other or use that only

**Review by prakharritik — CHANGES_REQUESTED** [review]

**Review by prakharritik — APPROVED** [review]

File: Frontend/ui-customer/src/Screens/RideBookingFlow/RiderRideCompletedCard/View.purs [hunk 5]
```diff
@@ -315,8 +367,168 @@ bottomCardView state push =
...
+customerIssuePopupView  push config = 
+  let 
+    cardsArray = config.customerIssueCards
+    firstCard  = head cardsArray 
+    title      = maybe "" (\x -> x.title) firstCard
+    selectedYes = maybe Nothing (\x -> x.selectedYes) firstCard
+    yestext    = maybe "" (\x -> x.yesText) firstCard
+    notext     = maybe "" (\x -> x.noText) firstCard
+    subtitle   = maybe "" (\x -> x.subTitle) firstCard
+    _ = spy "Here with length == 1" config
```
Comment (prakharritik) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/RiderRideCompletedCard/View.purs:443]: Remove spy

**Review by prakharritik — APPROVED** [review]

**Pull request merged**